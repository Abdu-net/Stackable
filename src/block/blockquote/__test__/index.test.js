import { edit, save } from '../index'

describe( 'Blockquote Block', () => {
	test( 'block edit matches snapshot', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
				color: '#424242',
				borderColor: '#2091e1',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( 'block save matches snapshot', () => {
		const wrapper = save( {
			attributes: {
				text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
				color: '#424242',
				borderColor: '#2091e1',
			},
		} )

		expect( wrapper ).toMatchSnapshot()
	} )
} )
