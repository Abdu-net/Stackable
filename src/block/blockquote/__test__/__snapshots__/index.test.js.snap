// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockquote block saved block HTML shouldn't change 1`] = `
<BlockContentProvider
  innerBlocks={Array []}
>
  <blockquote
    className="wp-block-ugb-blockquote ugb-blockquote ugb-blockquote--v2 ugb--background-opacity-2 ugb-blockquote--design-design ugb-blockquote--small-quote className"
    style={
      Object {
        "--quote-color": "#123456",
        "--ugb-background-color": undefined,
        "backgroundAttachment": undefined,
        "backgroundColor": undefined,
        "backgroundImage": undefined,
        "borderRadius": undefined,
      }
    }
  >
    <div
      className="ugb-content-wrapper"
    >
      <Unknown
        style={
          Object {
            "fill": "#123456",
            "height": 35,
            "width": 35,
          }
        }
      />
      <Unknown
        className="ugb-blockquote__text"
        format="string"
        style={
          Object {
            "color": "#123456",
          }
        }
        tagName="p"
        value="text"
      />
    </div>
  </blockquote>
</BlockContentProvider>
`;

exports[`Blockquote block saved default block HTML shouldn't change 1`] = `
<BlockContentProvider
  innerBlocks={Array []}
>
  <blockquote
    className="wp-block-ugb-blockquote ugb-blockquote ugb-blockquote--v2 ugb--background-opacity-5 ugb-blockquote--design-plain"
    style={
      Object {
        "--quote-color": undefined,
        "--ugb-background-color": undefined,
        "backgroundAttachment": undefined,
        "backgroundColor": undefined,
        "backgroundImage": undefined,
        "borderRadius": undefined,
      }
    }
  >
    <div
      className="ugb-content-wrapper"
    >
      <Unknown
        style={
          Object {
            "fill": "",
            "height": 70,
            "width": 70,
          }
        }
      />
      <Unknown
        className="ugb-blockquote__text"
        format="string"
        style={
          Object {
            "color": undefined,
          }
        }
        tagName="p"
        value="Description for this block. Use this space for describing your block. Any text will do. Description for this block. You can use this space for describing your block. Description for this block. Use this space for describing your block. Any text will do. Description for this block. You can use this space for describing your block."
      />
    </div>
  </blockquote>
</BlockContentProvider>
`;
